jobs/JGLOBAL_FORECAST

```diff
@@ -61,13 +61,13 @@ fi
 if [[ ${DO_OCN} == "YES" ]]; then
   YMD=${PDY} HH=${cyc} generate_com -rx COM_MED_RESTART COM_OCEAN_RESTART COM_OCEAN_INPUT \
     COM_OCEAN_HISTORY COM_OCEAN_ANALYSIS
-  RUN=${CDUMP} YMD="${gPDY}" HH="${gcyc}" generate_com -rx \
+  RUN=${rCDUMP} YMD="${gPDY}" HH="${gcyc}" generate_com -rx \
     COM_OCEAN_RESTART_PREV:COM_OCEAN_RESTART_TMPL
 fi
 
 if [[ ${DO_ICE} == "YES" ]]; then
   YMD=${PDY} HH=${cyc} generate_com -rx COM_ICE_HISTORY COM_ICE_INPUT COM_ICE_RESTART
-  RUN=${CDUMP} YMD="${gPDY}" HH="${gcyc}" generate_com -rx \
+  RUN=${rCDUMP} YMD="${gPDY}" HH="${gcyc}" generate_com -rx \
```

Comment:
This change will cause the current forecast-only tests to fail.
A change is needed in https://github.com/NOAA-EMC/global-workflow/blob/8940adddfe9d21189740e71487603fb2acee2336/scripts/exglobal_stage_ic.sh#L76 to place the ice restarts in `gdas.${gPDY}/${gcyc}`